### PR TITLE
Address server refactoring in lookout-sdk-ml

### PR DESCRIPTION
**Pull request by zurk**
Goes after https://github.com/src-d/lookout-sdk-ml/pull/66 merged and new `lookout-sdk-ml` package release.

### CODE REVIEW

File: lookout/style/format/benchmarks/evaluate_smoke.py [hunk 1]
```diff
@@ -300,7 +299,7 @@ def evaluate_smoke_entry(inputpath: str, reportdir: str, database: str, bblfsh:
     start_time = time.time()
     report_filename = os.path.join(reportdir, "report.csv")
     log = logging.getLogger("evaluate_smoke")
-    port = server.find_port()
+    lookout = LookoutSDK()
```
Comment (vmarkovtsev) [inline lookout/style/format/benchmarks/evaluate_smoke.py]:
```suggestion
    lookout_sdk = LookoutSDK()
```

(we've got `lookout` package)

File: lookout/style/format/benchmarks/expected_vnodes_number.py [hunk 2]
```diff
@@ -46,15 +46,15 @@ def _restart_bblfshd(first_run: bool=False) -> str:
...
+def get_vnodes_number(repository: str, from_commit: str, to_commit: str, lookout: LookoutSDK,
                       bblfsh: Optional[str]) -> int:
     """
     Calculate the number of expected vnodes number for a repository.
 
     :param repository: URL of repository.
     :param from_commit: Hash of the base commit.
     :param to_commit: Hash of the head commit.
-    :param port: Port for FormatAnalyzer.
+    :param lookout: LookoutSDK instance to query analyzer.
```
Comment (vmarkovtsev) [inline lookout/style/format/benchmarks/expected_vnodes_number.py]:
```suggestion
    :param lookout_sdk: LookoutSDK instance to query analyzer.
```

File: lookout/style/format/benchmarks/expected_vnodes_number.py [hunk 3]
```diff
@@ -96,9 +96,8 @@ def calc_expected_vnodes_number_entry(input: str, output: str, runs: int) -> Non
     handler = logging.handlers.RotatingFileHandler(output + ".errors")
     handler.setLevel(logging.ERROR)
     log.addHandler(handler)
-    if not server.exefile.exists():
-        server.fetch()  # download executable
-    port = server.find_port()
+
+    lookout = LookoutSDK()
```
Comment (vmarkovtsev) [inline lookout/style/format/benchmarks/expected_vnodes_number.py]:
```suggestion
    lookout_sdk = LookoutSDK()
```

**Comment by vmarkovtsev** [comment]
@zurk No longer blocked, 0.15 has been published

**Comment by zurk** [comment]
PR is rebased, tests pass. @vmarkovtsev shall we? :)

**Pull request merged**